web/profiles/custom/yalesites_profile/modules/custom/ys_core/config/install/ys_core.settings.yml

```diff
@@ -0,0 +1,6 @@
+ys_core:
+  social_facebook_link: ''
+  social_instagram_link: ''
+  social_twitter_link: ''
+  social_youtube_link: ''
+  social_weibo_link: ''
```

Comment:
By default, configuration will be wiped out when changes are pushed to the downstream site. That is, the links stored in the database will be repalced on config-import each time we deploy changes. Please at this configuration file to the list of configs to ignore. This is only a partial solution and we will need additional modules in the future to support these per-site configs. admin/config/development/configuration/ignore